axelrod/moran.py

```diff
@@ -1,33 +1,39 @@
 """Implementation of the Moran process on Graphs."""
 
-from collections import Counter
 import random
+from collections import Counter
+from typing import Callable, List, Optional, Set, Tuple
 
 import matplotlib.pyplot as plt
 import numpy as np
 
-from axelrod import DEFAULT_TURNS, Player, Game
+from axelrod import DEFAULT_TURNS, Game, Player
+
 from .deterministic_cache import DeterministicCache
-from .graph import complete_graph, Graph
+from .graph import Graph, complete_graph
 from .match import Match
 from .random_ import randrange
 
-from typing import List, Tuple, Set, Optional
 
-
-def fitness_proportionate_selection(scores: List) -> int:
+def fitness_proportionate_selection(
+    scores: List, fitness_function: Callable = None
+) -> int:
     """Randomly selects an individual proportionally to score.
 
     Parameters
     ----------
     scores: Any sequence of real numbers
+    fitness_function: A function mapping a score to a (non-negative) float
```

Comment:
Following @marcharper's suggestion can we change this to be `fitness_transformation` as the variable name? Similarly in `MoranProcess.__init__` and the docs :+1: :)